client/src/app/routes/Landing.scss

```diff
@@ -256,48 +208,254 @@
+                          position: absolute;
+                          top: 45%;
+                          left: 50%;
+                          transform: translate(-50%, -50%);
+
+                          @include font-size(16px);
+                        }
+                      }
+                    }
+                  }
+
+                  .label {
+                    margin-left: 8px;
+                    color: $gray-6;
+                    font-family: $inter;
+                    font-weight: 500;
+                    letter-spacing: 0.5px;
+
+                    @include font-size(12px, 18px);
+                  }
+                }
+
+                .filter--value {
+                  display: flex;
+                  margin: 0 8px;
+                  color: var(--expanded-color);
+                  font-family: $inter;
+
+                  @include font-size(14px, 21px);
+
+                  .input-plus {
+                    width: 94px;
+                    padding: 8px;
+                    border: 0;
+                    border-right: 0;
+                    border-radius: 6px 0 0 6px;
+                    background-color: var(--light-bg);
+                    box-shadow: 0 0 0 0;
+                    color: var(--expanded-color);
+                  }
+
+                  .select-plus {
+                    min-width: 20px;
+                    padding: 0 39px 0 11px;
+                    border: 0;
+                    border-left: 0;
+                    border-radius: 0 6px 6px 0;
+                    background-color: var(--light-bg);
+                    color: var(--expanded-color);
+
+                    &:focus {
+                      border-color: var(--light-bg);
+                    }
+                  }
+                }
+
+                label {
+                  display: block;
+                  padding: 12px 10px;
+                  color: var(--expanded-color);
+                  font-family: $inter;
+                  letter-spacing: 0.5px;
+
+                  @include font-size(14px, 21px);
+
+                  input {
+                    position: relative;
+                    width: 20px;
+                    height: 20px;
+                    margin-right: 18px;
+                  }
+                }
+              }
             }
 
             .feed-items {
               min-height: 440px;
-              padding: 30px;
-              background-color: #fbfcfe;
+              padding: 24px 0;
 
-              @include phone-down {
-                padding: 20px;
+              p {
+                color: var(--body-color);
               }
 
               .feed-item--header {
+                justify-content: space-between;
                 margin-bottom: 10px;
+                padding: 0 8px;
+
+                .label {
+                  @include font-size(12px, 18px);
 
-                .card--label {
                   min-width: 90px;
+                  color: $gray-6;
```

Comment:
Change color for dark theme from $gray-6 to $gray-5.

Reply:
Resolved